packages/rustworkx/meta.yaml

```diff
@@ -0,0 +1,26 @@
+package:
+  name: rustworkx
+  version: 0.17.0a3
+  top-level:
+    - rustworkx
+  tag:
+    - rust
+source:
+  url: https://github.com/IvanIsCoding/rustworkx/releases/download/v0.17.0a3/rustworkx-0.17.0a3.tar.gz
```

Comment:
Non-blocking; could we perhaps use the PyPI sdist here?

Reply:
I'd like to eventually, but the current status is:
* 0.16 will fail to build as it does not contain https://github.com/Qiskit/rustworkx/pull/1445
* https://github.com/Qiskit/rustworkx/pull/1447 is not yet reviewed nor merged
* We've never published a pre-release on PyPI, I think our release setup could work but  that is TBD. So assume 0.17 will take a couple of weeks to release

So publishing a "patched" alpha release to my fork seemed like a fair compromise. The moment I get a PyPI release I'll send a PR updating

Reply:
Sounds good to me, thanks for the info!